### DISCUSSION TITLE
Failed to config band1 40MHz for quectel-RM500QGL

### DISCUSSION THREAD

**Discussion by Zplusless** [srsran/srsRAN_Project#724] [General Help]
Hi community,

I'm trying to connect the Quectel RM500QGL to a gNB equipped with a USRP-B210. I have successfully set up a connection with band 3 at 30 MHz (config file: [usrp_fdd_n3_30MHz.yml.txt](https://github.com/user-attachments/files/16173029/usrp_fdd_n3_30MHz.yml.txt)). However, according to the official document of Quectel, the RM500QGL supports band 3 with a maximum of 30 MHz and band 1 with a maximum of 40 MHz.

![image](https://github.com/srsran/srsRAN_Project/assets/10062702/427b1a93-2fb3-47b0-b87a-c070122058ad)

I changed `dl_arfcn`, `srate`, and `band` in the config file (config file: [test_fdd_n1_40MHz.yml.txt](https://github.com/user-attachments/files/16173087/test_fdd_n1_40MHz.yml.txt)), but the RM500QGL failed to attach.

gnb log: [gnb.log](https://github.com/user-attachments/files/16173190/gnb.log)

I'm not sure if the modification is correct. If there are any problems, please point them out. Thanks in advance.

btw：Quectel RM500QGL and gNB are connected with SMA cable and spliter (following the [tutorial](https://kb.ettus.com/OAI_Reference_Architecture_for_5G_and_6G_Research_with_USRP)) like this: ![](https://kb.ettus.com/images/7/7a/Photo_of_system_3.JPG)

**Comment by herlesupreeth** [comment]
Can you change the freq back to band 3 and attach the modem and send me the F1AP pcap? I want to check the UE Capabilities information which is a better indication of what BWs UE supports at a particular band

**Reply by Zplusless** [reply]
Thank you for your reply. As requested, please find attached below the FIAP pcap files for n3_30MHz, n78_40MHz, and n78_50MHz:
[gnb-f1ap.zip](https://github.com/user-attachments/files/16174631/gnb-f1ap.zip)

**Reply by herlesupreeth** [reply]
Thank you. I see that your modem indeed support 40Mhz on Band 1. Can you please rerun the gNB and set all log levels to info and post it here? The logs you posted earlier has only FAPI logs in debug

**Reply by Zplusless** [reply]
Thanks!  I have tested n1 40MHz with all log level set to info. The command is `sudo gnb -c test_fdd_n1_40MHz.yml -c log_level_info.yaml` and the config files and the gnb.log are as follows:

[gnb-n1_40MHz-all_level_info.log](https://github.com/user-attachments/files/16179192/gnb-n1_40MHz-all_level_info.log)
[log_level_info.yaml.txt](https://github.com/user-attachments/files/16179194/log_level_info.yaml.txt)
[test_fdd_n1_40MHz_.yml.txt](https://github.com/user-attachments/files/16179198/test_fdd_n1_40MHz_.yml.txt)


Besides, I have two more questions:
1. Regarding ARFCN, can I set its value to any number that is divisible by 20 within its range? 
2. My wireshark cannot decode f1ap.pcap with "user encapsulation not handled: DLT=154".  How to solve it? Current I cannot read anything valuable from the pcap file.

**Reply by herlesupreeth** [reply]
> My wireshark cannot decode f1ap.pcap with "user encapsulation not handled: DLT=154". How to solve it? Current I cannot read anything valuable from the pcap file.

Here is the link to a guide to see F1AP packet - https://docs.srsran.com/projects/project/en/latest/user_manuals/source/outputs.html#f1ap

> Regarding ARFCN, can I set its value to any number that is divisible by 20 within its range?

I think so. Usually I pick the NR ARFCN at the center of the band

Thanks for the logs. I dont see UE even sending a PRACH, not sure whats wrong there. One thing I notice is the high Rx gain in the gNB configuration file

```
  tx_gain: 70                                                    # Transmit gain of the RF might need to adjusted to the given situation.
  rx_gain: 60                                                    # Receive gain of the RF might need to adjusted to the given situation. 
```

**Reply by Zplusless** [reply]
Thanks! I'll modify the config file and feed back with any update.

**Reply by pgawlowicz** [reply]
@Zplusless any update on this issue?

**Reply by Zplusless** [reply]
Sadly, Not solved yet.
